packages/pharos/src/components/link/pharos-link.scss

```diff
@@ -95,6 +95,11 @@
   color: var(--pharos-color-interactive-secondary);
 }
 
+:host([visitable]) #link-element:visited {
```

Comment:
The name visitable implies that other links without this attribute are **not** visitable, which is probably a source of confusion. I discussed this previously with @chrisjbrown but it looks like we didn't reach a conclusion on another name; might we call this something like `indicate-visited`?

Reply:
yeah we didn't land on a different name
`indicate-visited` sounds good to me

Reply:
Updated in [ab9058a](https://github.com/ithaka/pharos/pull/409/commits/ab9058a239f6a6b1f2ce37a40e461ca48679f1a9)